.travis.yml

```diff
@@ -11,7 +11,7 @@ matrix:
   include:
     - php: '7.0'
     - php: '7.1'
-      env: coverage=1 lint=1
+      env: coverage=0 lint=1
```

Comment:
We need to restore this later, but I've no idea how to fix the problem for now.

Reply:
What is the problem with the coverage?

Reply:
We reach the `ulimit` defined by Travis... Too many open files.

Reply:
can't we `ulimit -n 4000` before install? It happens on my local computer too when testing the `Extension`.

Reply:
Unfortunately no, Travis disallow to increase the default value.